packages/react-player/src/controls.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright 2023 Design Barn Inc.
+ */
+
+import { PlayerState } from 'common';
+import React, { useContext, useMemo, useState } from 'react';
+
+import { DotLottieContext } from './react-player';
+
+const AVAILABLE_BUTTONS = ['play', 'stop', 'loop'] as const;
+
+interface ControlsProps extends React.HTMLAttributes<HTMLDivElement> {
+  buttons?: Array<typeof AVAILABLE_BUTTONS[number]>;
+  show?: boolean;
+}
+
+export const Controls: React.FC<ControlsProps> = ({ buttons = AVAILABLE_BUTTONS, ...props }) => {
+  const { currentState, dotLottiePlayer, frame, loop, seeker } = useContext(DotLottieContext);
```

Comment:
From what I understand, we're using react context for managing state. However, I've observed some state duplications like `isLoop` state, leading to multiple sources of truth for states. I'm curious if it would be possible to utilize context as the sole source of truth for our states.